JuliaInterface/src/JuliaInterface.h

```diff
@@ -71,17 +71,6 @@ Obj NewJuliaFunc(jl_function_t * f, int autoConvert);
 // from the julia value pointer v.
 Obj NewJuliaObj(jl_value_t *);
 
-// Func_JuliaFunction( NULL, string )
-//
-// Returns the function with name <string> from the Julia `Main` module.
-Obj Func_JuliaFunction(Obj self, Obj string);
-
-// Func_JuliaFunctionByModule( NULL, function_name, module_name )
-//
-// Returns the function with name <function_name> from
-// the Julia module with name <module_name>.
-Obj Func_JuliaFunctionByModule(Obj self, Obj function_name, Obj module_name);
-
```

Comment:
Actually, to the contrary, I'd remove *all* those `Func*` declarations from the header. They are for interfacing with the GAP interpreter only, and exposing them to external use is IMHO bad design.

Instead, if external code needs to access some functionality, then I'd recommend to add specific functions to expose the functionality in a clean and documented way.

Just compare this to what you've been suggesting / asking for regarding an "official libgap API": Right now, everything in the GAP kernel is the "kernel API" (and it is therefore very unstable in terms of compatibility, as no promises are made). Instead, you are asking GAP to provide an explicit, stable API. (And if GAP was starting from scratch, I might also be arguing for it). Why treat this kernel extension differently?

Reply:
That is true. We should maybe keep this in mind, because some stuff from the header might be interesting for further kernel extensions, as in `JuliaExperimental`

Reply:
Well, I'd rather do this: remove everything from the header, now. Then, add them back on request resp. if they are needed. Since we know all people working on this, that shouldn't be a major problem, should it?

Reply:
Were "everything" means: "Every Func_FOO", but of course not stuff that's already used by JuliaExperimental"

Reply:
Yep, that is a good idea. I will do this, maybe recovering some of the comments.